.github/workflows/dependabotautomerge.yml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: auto-merge
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
```

Comment:
any idea whether dependabot supports "batched" PRs? Instead of having a single PR per version update, we would have one per week or so, which seems reasonable for minor/patch updates.
The downside is that if an update breaks builds, we'll have to go thru each update to figure out which is the culprit.

Reply:
found https://github.com/dependabot/dependabot-core/issues/2233